src/common/scheduler/policy/evaluation_result.go

```diff
@@ -0,0 +1,15 @@
+package policy
+
+//EvaluationResult is defined to carry the policy evaluated result.
+//
+//Filed 'Result' is optional.
+//Filed 'Error' is optional
+//
+type EvaluationResult struct {
```

Comment:
Only the current daily policy implementation does not handle the errors and results. Other policies might need this channel to pass the evaluation result to the tasks and send async errors to the upper module. We need to consider these scenarios.

I have considered your suggestion. Return separate chan for result and error.  I just don't want to make the return signature too complicated. If you think separate ones are better, I can refine it.